### DISCUSSION TITLE
What new sub-workflows should be added to atlas?

### DISCUSSION THREAD

**Discussion by SilasK** [metagenome-atlas/atlas#468] [Ideas]
Give us your ideas of what you would like to add to metagenome-atlas. 

Please use a new chunk per idea and comment or upvote ideas.
Ideas that are mentioned elsewhere don't need to be repeated? 

Solid ideas will be transformed to feature requests.

**Comment by SilasK** [comment]
@alienzj you want to tell us about your ideas?

**Reply by alienzj** [reply]
Yes.
